src/copyright/ui/ajax-copyright-hist.php

```diff
@@ -100,10 +100,17 @@ function Output()
     else if($action=="update" || $action=="delete" || $action=="undo")
     {
       $id = GetParm("id", PARM_STRING);
-      list($upload, $item, $hash, $type) = explode(",", $id);
+      $getEachID = array_filter(explode(",", trim($id, ',')));
```

Comment:
`array_filter` will also drop `"0"`, which casts to `false`. (e.g. `array_filter(array("", "0", "1")) == array("2" => "0")`)

Reply:
I think 0  is not accepted for every value in this ($upload, $item, $hash, $type).

EX: When uploadId is 0 it is an invalid input anyway.

Reply:
please make that explicit. This will make our code base a little bit cleaner and is best practice in PHP.

Reply:
I was googleing it for curiosity ... but what would be good programming practice in this case?